src/client/app/components/unit/CreateUnitModalComponent.tsx

```diff
@@ -111,180 +162,274 @@ export default function CreateUnitModalComponent() {
+													disabled={
+														(state.typeOfUnit == UnitType.meter ||
+															state.suffix != '') &&
+														key != DisplayableType.none
+													}
+												>
+													{translate(`DisplayableType.${key}`)}
+												</option>
+											);
+										})}
+									</Input>
+									<FormFeedback>
+										{state.displayable !== DisplayableType.none && state.typeOfUnit == UnitType.meter ? (
+											<FormattedMessage id="error.displayable.meter" />
+										) : (
+											<FormattedMessage id="error.displayable.suffix.input" />
+										)}
+									</FormFeedback>
+								</FormGroup>
+							</Col>
+							{/* Preferred display input */}
+							<Col>
+								<FormGroup>
+									<Label for="preferredDisplay">
+										{translate('unit.preferred.display')}
+									</Label>
+									<Input
+										id="preferredDisplay"
+										name="preferredDisplay"
+										type="select"
+										onChange={e => handleBooleanChange(e)}
+									>
+										{Object.keys(TrueFalseType).map(key => {
+											return (
+												<option value={key} key={key}>
+													{translate(`TrueFalseType.${key}`)}
+												</option>
+											);
+										})}
+									</Input>
+								</FormGroup>
+							</Col>
+						</Row>
+						<Row xs="1" lg="2">
+							{/* Seconds in rate input */}
+							<Col>
+								<FormGroup>
+									<Label for="secInRate">{translate('unit.sec.in.rate')}</Label>
+									<Input
+										id="secInRate"
+										name="secInRate"
+										type="select"
+										onChange={e => handleStandardNumberChange(e)}
+										value={rate}>
+										{Object.entries(LineGraphRates).map(
+											([rateKey, rateValue]) => (
+												<option value={rateValue * 3600} key={rateKey}>
+													{translate(rateKey)}
+												</option>
+											)
+										)}
+										<option value={CUSTOM_INPUT}>
+											{translate('custom.value')}
+										</option>
+									</Input>
+									{showCustomInput && (
+										<>
+											<Label for="customRate">
+												{translate('unit.sec.in.rate.enter')}
+											</Label>
+											<Input
+												id="customRate"
+												name="customRate"
+												type="number"
+												value={customRate}
+												min={1}
+												invalid={!customRateValid(customRate)}
+												onChange={e => handleCustomRateChange(e)}
+												onKeyDown={e => { handleEnter(e.key); }}
+											/>
+										</>
+									)}
+									<FormFeedback>
+										<FormattedMessage id="error.greater" values={{ min: '0' }} />
+										{translate('and')}{translate('an.integer')}
+									</FormFeedback>
+								</FormGroup>
+							</Col>
+							{/* Suffix input */}
+							<Col>
+								<FormGroup>
+									<Label for="suffix">{translate('unit.suffix')}</Label>
+									<Input
+										id="suffix"
+										name="suffix"
+										type="text"
+										autoComplete="off"
```

Comment:
Not sure, since suffix is a unit type I think autocomplete might serve as quality of life for users who don't want to have to retype for meters that share the same unit. Then again, I'm not familiar with the purpose of suffix. I would love some additional info on how it works so I can give more informed feedback.

Reply:
Suffix is a string that is the end of a special type of unit's name. It seems unlikely that it would be reused or match a standard used string. I think it is probably best not to allow autocomplete in this case.

Reply:
removed as requested